data/sql/updates/pending_db_world/rev_1586023897838741800.sql

```diff
@@ -0,0 +1,42 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1586023897838741800');
+
+-- Delete queststarter for creature 20482
+DELETE FROM `creature_queststarter` WHERE  `id`=20482 AND `quest`=10975;
+DELETE FROM `creature_queststarter` WHERE  `id`=20482 AND `quest`=10976;
+DELETE FROM `creature_queststarter` WHERE  `id`=20482 AND `quest`=10977;
+DELETE FROM `creature_queststarter` WHERE  `id`=20482 AND `quest`=10981;
+
+-- Update queststarter for quest 10982 to 22919 (was previously set to 20482)
+UPDATE `creature_queststarter` SET `id`='22919' WHERE  `id`=20482 AND `quest`=10982;
```

Comment:
Personally I would do a DELETE and INSERT for this one. To me, it does not feel right to update a key...

Reply:
The Update is fine here